### ISSUE TITLE
Combobox inaccessible using VoiceOver

### ISSUE THREAD

**Issue by wkeese** [carbon-design-system/carbon#6587]
Combobox can't be used via VoiceOver on iOS.

## Environment

> Operating system

iOS

> Browser

Safari

> Assistive technology used to verify

VoiceOver

## Detailed description

> What version of the Carbon Design System are you using?

Carbon Components React v7.16

> What did you expect to happen?

Should be able to type in filter text and open the dropdown using VoiceOver swipe and double-click gestures.

> What happened instead?

AFAICT you can neither type in filter text nor open the dropdown, because the VoiceOver cursor only goes to the wrapper around the whole Combobox, but not to the input nor to the caret.

## Steps to reproduce the issue

1. Navigate to https://react.carbondesignsystem.com/iframe.html?id=combobox--default.
2. Triple-click home button to turn on VoiceOver.
3. Try to open dropdown or filter just using swipe gestures and double-click.

Note that you can open the dropdown by directly clicking on the chevron, but if you are able to see the chevron then you don't need VoiceOver in the first place.

> Please create a reduced test case in CodeSandbox

https://react.carbondesignsystem.com/iframe.html?id=combobox--default

## Additional information

This is the only place that the VoiceOver cursor goes by swipe left/right:

<img width="313" alt="Screen Shot 2020-07-30 at 7 42 38" src="https://user-images.githubusercontent.com/69599/88861192-6869ff00-d238-11ea-8b9e-d8c3b847fee0.png">

**Comment by dakahn** [comment]
Unable to recreate this bug using iOS14 and Safari on mobile with latest Carbon here https://react.carbondesignsystem.com/iframe.html?id=components-combobox--combobox. 

Steps to recreate
1. With the Storybook canvas focused swipe left until the combobox is given focus by the virtual cursor and reads "Combobox title, filter elipsis, combobox listbox popup, collapsed double tap to expand"
2. Doube tap the screen to expand the listbox and open the iOS keybaord

Let me know if that doesn't work for you and I can try to assist 👍🏽

**Comment by wkeese** [comment]
Cool, it's working for me too now, must have been fixed with other combobox changes.
